README.md

```diff
@@ -107,6 +109,17 @@ If you are using an alias like `k="kubecolor"`, then just change above like:
 complete -o default -F __start_kubectl k
 ```
 
+#### Zsh
+
+For zsh make sure this lines are present in your .zshrc file:
```

Comment:
Please fix this for consistency with another part.

```suggestion
For zsh make sure these lines are present in your zsh config file.
```

Reply:
Updated these...